src/implementations/twig/components/link/link.story.js

```diff
@@ -26,86 +25,125 @@ const withParagraph = (story) => {
   return typeof demo === 'string' ? storyAsString(demo) : storyAsNode(demo);
 };
 
-const iconsList = {};
-iconsList.none = '';
-
-uiIcons.forEach((icon) => {
-  iconsList[icon] = icon;
-});
+const iconsList = uiIcons.reduce((a, b) => {
+  a[b] = b;
+  return a;
+}, {});
+
+const getArgTypes = (data) => {
+  return {
+    label: {
+      name: 'label',
+      type: { name: 'string', required: true },
+      defaultValue: data.link.label,
+      description: 'The main label of the button',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Link',
+      },
+      control: {
+        type: 'text',
+      },
+    },
+    aria_label: {
```

Comment:
I don't think we need to keep this control for aria-label; that's not something directly visible

Reply:
i've reproduced the knobs we have in ecl-v2, where the aria label knob is present. Anyway, ok.